actions/deploy-renku/deploy_tmp_notebooks.py

```diff
@@ -187,16 +192,95 @@ def check_notebooks_chart_version():
         .stdout.decode("utf-8")
         .split("\n")
     )
+    renku_notebooks_version = None
     for dependency in chart_dependencies:
         if "renku-notebooks" in dependency:
             [_, renku_notebooks_version, renku_chart_repo, _] = dependency.split("\t")
-    return renku_notebooks_version
+    if renku_notebooks_version:
+        return renku_notebooks_version
+    raise RuntimeError("Unable to determine the renku-notebooks version.")
+
+
+def deploy_tmp_notebooks(
+    release_name,
+    renku_namespace,
+    renku_notebooks_version,
+    tmp_namespace,
+    cleanup=True,
+    deploy=True,
+    extra_values=None,
+    kube_context=None,
+    local_path=None,
+):
+    """Do the anonymous sessions deployment."""
+    kube_context = kube_context or k8s.config.list_kube_config_contexts()[1]["name"]
+
+    # Get an k8s api client from the specified local context
+    k8s_api_client = k8s.client.CoreV1Api(
+        k8s.client.ApiClient(
+            k8s.config.load_kube_config(
+                config_file=os.environ["KUBECONFIG"],
+                context=kube_context
+                or k8s.config.list_kube_config_contexts()[1]["name"],
+            )
+        )
+    )
+
+    # Create secondary namespace using the k8s python client,
+    # skip if it already exists.
+    create_tmp_namespace(k8s_api_client, tmp_namespace)
+
+    # Copy the secret using the k8s python client,
+    # skip if it already exists.
+    copy_secret(k8s_api_client, renku_namespace, tmp_namespace)
+
+    # Create a values file for the secondary renku-notebooks deployment
+    renku_values = get_values(release_name, kube_context, renku_namespace)
+    tmp_values_path = make_tmp_values(renku_values, renku_namespace)
+
+    # Deploy through the helm CLI
+    helm_command = [
+        "helm",
+        "upgrade",
+        "--install",
+        f"{renku_namespace}-tmp-notebooks",
+        "--namespace",
+        f"{tmp_namespace}",
+        "-f",
+        f"{tmp_values_path}",
+        "--timeout",
+        "1800s",
+        "--kube-context",
+        f"{kube_context}",
+    ]
+
+    # check if the notebooks version is a local path
+    if local_path:
+        helm_command += ["renku-notebooks", str(local_path)]
```

Comment:
```suggestion
        helm_command += [str(local_path)]
```
I think when providing a local chart you don't give the name of the chart to that command (take `helm upgrade --install andreas-renku ./renku` for example).

Reply:
hmm I suppose you're right though I was pretty sure I tested this :)